x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/field_browser.tsx

```diff
@@ -153,53 +137,22 @@ const FieldsBrowserComponent: React.FC<Props> = ({
     [onSearchInputChange]
   );
 
-  const scrollViewsAndFocusInput = useCallback(() => {
-    scrollCategoriesPane({
-      containerElement: containerElement.current,
-      selectedCategoryId,
-      timelineId,
-    });
-
-    // always re-focus the input to enable additional filtering
-    focusSearchInput({
-      containerElement: containerElement.current,
-      timelineId,
-    });
-  }, [selectedCategoryId, timelineId]);
-
-  useEffect(() => {
-    scrollViewsAndFocusInput();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [selectedCategoryId, timelineId]);
-
-  const onKeyDown = useCallback(
```

Comment:
Accessibility controls no longer needed, `EuiInMemoryTable` brings them built-in